box-browse-sdk/src/main/java/com/box/androidsdk/browse/uidata/LoaderDrawable.java

```diff
@@ -128,8 +170,19 @@ public static ThumbnailTask create(final BoxRequestsFile.DownloadThumbnail reque
     }
 
 
+    /**
+     * The interface Image ready listener.
+     */
```

Comment:
Callback interface for signifying that the thumbnail has been loaded into the imageview.